projects/schematics/src/add-spartacus/index.ts

```diff
@@ -286,6 +286,42 @@ function increaseBudgets(options: SpartacusOptions): Rule {
   };
 }
 
+/**
+ * Checks if the app has an app configuration file and uses standalone components by default.
+ *
+ * @param options - The Spartacus options.
+ * @returns A Rule function that checks if the app has an app configuration file.
+ */
+function hasAppConfigFile(options: SpartacusOptions): Rule {
```

Comment:
1. hasXxx() function name suggests it returns boolean. it returns Tree.
2. hasAppConfigFile is the function name. but the implementation doesn't check it. it checks whether it has app.module. Maybe we should change the function name?